.travis.yml

```diff
@@ -25,7 +25,7 @@ install:
 
 jobs:
   include:
-    - stage: test
```

Comment:
Use a simpler one word for the stage (less typo prone), Travis already lower cases too 

You can use the longer description as a name
https://docs.travis-ci.com/user/build-stages/#naming-your-build-stages